### PR TITLE
SYNPY-966 empty structure for the core package

### PR DESCRIPTION
**Pull request by kimyen**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# This section allows us to run
+# python setup.py test
+[aliases]
+test=pytest
+
+# This section defines setup for tests
+[tool:pytest]
+addopts = --verbose
+python_files = tests/*/*.py
```
Comment (zimingd) [inline setup.cfg]: check wildcards to ensure that all test will be run

Comment (zimingd) [reply setup.cfg]: ensure that it is not reliant on there being exactly 2 levels of folders

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    # basic
+    name='spccore',
+    version=about["__version__"],
+    packages=setuptools.find_packages(exclude=["tests", "tests.*"]),
+
+    # requirements
+    python_requires='>=3.5.*',
+    install_requires=[
+        'requests>=2.21.0',
+        'keyring==12.0.2',
```
Comment (zimingd) [inline setup.py]: consider upgrading to latest version if they are stable. maybe look into using other library.

File: setup.cfg (hunk 1 above)
Comment (kimyen) [reply setup.cfg]: Pytest uses glob-style file pattern: https://docs.pytest.org/en/latest/reference.html#confval-python_files
It looks like we needs double stars: https://facelessuser.github.io/wcmatch/glob/

**Pull request merged**